Formula/boost.rb

```diff
@@ -20,6 +20,14 @@ class Boost < Formula
   # patch derived from https://github.com/boostorg/graph/commit/1d5f43d
   patch :DATA
 
+  # Fix auto-pointer registration in 1.60
+  # https://github.com/boostorg/python/pull/59
+  # patch derived from https://github.com/boostorg/python/commit/f2c465f
+  patch do
+    url "https://gist.githubusercontent.com/mcs07/cf65084c7b99a64e8cac2fed7a85e23b/raw/78e3fc6e8cfabd6e0a44dc40775f3187690387de/boost-1.60.diff"
```

Comment:
Could you possibly submit this to our [patches repo](https://github.com/Homebrew/patches) and use the commit link (once merged) from over there?

We're trying to centralise patches to a degree as an element of protection in case a user accidentally deletes the wrong gist on their account/etc and suddenly it starts falling over for everyone.